test/verify/check-system-info

```diff
@@ -947,6 +947,12 @@ password=foobar
         b.wait_text("#crypto-policy-button", "FIPS")
         self.assertEqual(m.execute("cat /proc/sys/crypto/fips_enabled").strip(), "1")
 
+        m.execute(cmd + " --set DEFAULT")
+        b.wait_text("#inconsistent_crypto_policy", "Crypto policy is inconsistent")
+        m.execute(cmd + " --set FIPS:OSPP")
+        b.wait_text("#crypto-policy-button", "Fips:ospp")
+        self.assertFalse(b.is_present("#inconsistent_crypto_policy"))
```

Comment:
Ah, for this case a wait_present() would actually be correct, the `#inconsistent-crypto-policy` object actually was present before, and with react the state update would be async. In the previous version of the patch, the "inconsistent" was not checked to be present.

So, one more round with wait_not_present here, and it's good!